### PR TITLE
[fix](terminal) remove echo database passwd

### PR DESCRIPTION
**Pull request by zbtzbtzbt**
# Proposed changes
echo password is dangerous, we should delete it.

echo_passwd

## Problem summary

1. remove echo passwd
2. add some necessary information (e.g. add timer for data load)
3. remove some redundant code

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
5. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
6. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zbtzbtzbt** [comment]
use shfmt to format.
e.g.
`shfmt  -w tools/ssb-tools/bin/load-ssb-data.sh`

File: tools/ssb-tools/bin/gen-ssb-data.sh [hunk 1]
```diff
@@ -126,7 +126,7 @@ date
 cd -
 
 # move data to $SSB_DATA_DIR
-mv "${SSB_DBGEN_DIR}"/*.tbl* "${SSB_DATA_DIR}/"
+mv "${SSB_DBGEN_DIR}"*.tbl* "${SSB_DATA_DIR}/"
```
Comment (adonis0147) [inline tools/ssb-tools/bin/gen-ssb-data.sh:129]: `"${SSB_DBGEN_DIR}"*.tbl*` looks weird. I think the original one is better. Maybe we should remove the `/` from the assignment of `SSB_DBGEN_DIR` in [#L32](https://github.com/apache/doris/blob/f8f4cba64f8e75910b0a447c1813a2203e5c99b4/tools/ssb-tools/bin/gen-ssb-data.sh#L32).

File: tools/ssb-tools/bin/gen-ssb-data.sh [hunk 2]
```diff
@@ -126,7 +126,7 @@ date
...
 
 # check data
-du -sh "${SSB_DATA_DIR}"/*.tbl*
+du -sh "${SSB_DATA_DIR}"*.tbl*
```
Comment (adonis0147) [inline tools/ssb-tools/bin/gen-ssb-data.sh:132]: Ditto

File: tools/ssb-tools/bin/gen-ssb-data.sh (hunk 1 above)
Comment (zbtzbtzbt) [reply tools/ssb-tools/bin/gen-ssb-data.sh:129]: ok, good idea, Thanks for your suggestion.

**Review by adonis0147 — DISMISSED** [review]
LGTM

**Comment by adonis0147** [comment]
Hi @zbtzbtzbt , would you like to rebase the branch master to re-trigger the checks?

**Comment by zbtzbtzbt** [comment]
> Hi @zbtzbtzbt , would you like to rebase the branch on master to re-trigger the checks?

ok

**Review by adonis0147 — APPROVED** [review]
LGTM

**Review by jackwener — APPROVED** [review]
LGTM

**Pull request merged**